QUEST_LV_0100.tsv

```diff
@@ -6249,7 +6249,7 @@ QUEST_LV_0100_20150317_006244	Cut Bramble's Roots.
 QUEST_LV_0100_20150317_006245	Approached the Bramble's roots but Gaigalas appeared. You must defeat Gaigalas before destroying Bramble's roots.
 QUEST_LV_0100_20150317_006246	Talk to Follower Jurga
 QUEST_LV_0100_20150317_006247	You cut off Bramble's roots. Return and talk to Follower Jurga.
-QUEST_LV_0100_20150317_006248	Defeat Gaigalas guarding Bramble's Root
+QUEST_LV_0100_20150317_006248	Defeat Gaigalas
```

Comment:
This should not be removed. It's different than just "Defeat". There are many instances where it says more than "Defeat x". Like in the tutorial with Rocktortuga. 

This also counts for the Molich line.